modules/core/04-channel/keeper/upgrade.go

```diff
@@ -208,12 +208,70 @@ func (k Keeper) AbortUpgrade(ctx sdk.Context, portID, channelID string, err erro
 	return nil
 }
 
+// ChanUpgradeAck is called by a module to accept the ACKUPGRADE handshake step of the channel upgrade protocol.
```

Comment:
godoc is from the spec? I think it could be improved. Maybe we can indicate that this function should only be called by the core IBC msg server?

Reply:
Added that it should only be called by the core IBC msg server. If you have any suggestions for better godoc feel free to throw 'em down :)